src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Person } from '../../types/Person';
+
+type Props = {
+  people: Person[];
+  onSelected?: (person: Person) => void;
+};
+
+export const PeopleList: React.FC<Props> = React.memo(
+  ({ people, onSelected = () => {} }) => {
+    return (
+      <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+        <div className="dropdown-content">
+          {people.map(person => (
+            <div
+              className="dropdown-item"
+              data-cy="suggestion-item"
+              key={person.name}
```

Comment:
Using `person.name` as a key might cause issues if there are duplicate names in the `people` array. It's recommended to use a unique identifier, such as an `id`, if available, to ensure that each key is unique.